QUICKSTART-ECS.md

```diff
@@ -0,0 +1,193 @@
+or the permissions specified in the [Amazon ECS First Run Wizard Permissions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/security_iam_id-based-policy-examples.html#first-run-permissions) IAM policy example.
+
+You'll also need [aws-cli](https://aws.amazon.com/cli/) set up to interact with AWS.
+
+
+## Create a cluster
+
+An Amazon ECS cluster is a logical grouping of tasks, services, and container instances.
+For more information about clusters, see
+[Amazon ECS clusters](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/clusters.html).
+
+You can create a cluster with the AWS CLI as follows:
+
+```
+aws ecs --region us-west-2 create-cluster --cluster-name bottlerocket
+```
+
+> Note: The command above and subsequent examples include the AWS region, so change it from `us-west-2` if you operate in another region.
+
+## Finding an AMI
+
+You can either use an official AMI provided by Amazon or build an AMI yourself using
+[our guide](https://github.com/bottlerocket-os/bottlerocket/blob/develop/BUILDING.md).
+
+The official AMI IDs are stored in
+[public SSM parameters](https://docs.aws.amazon.com/systems-manager/latest/userguide/parameter-store-public-parameters.html).
+To find the `aws-ecs-1` variant for the `x86_64` architecture in the `us-west-2` region, you can use the following command:
+
+```
+aws ssm get-parameter --region us-west-2 --name "/aws/service/bottlerocket/aws-ecs-1/x86_64/latest/image_id" --query Parameter.Value --output text
+```
+
+If you have `jq` and would like a bit more information, try this:
+
+```
+aws ssm get-parameters --region us-west-2 \
+   --names "/aws/service/bottlerocket/aws-ecs-1/x86_64/latest/image_id" \
+           "/aws/service/bottlerocket/aws-ecs-1/x86_64/latest/image_version" \
+   --output json | jq -r '.Parameters | .[] | "\(.Name): \(.Value) (updated \(.LastModifiedDate | gmtime | strftime("%c")) UTC)"'
+```
+
+You can replace the architecture (`x86_64`) and region (`us-west-2`) to look for other images.
+If you know a specific Bottlerocket version you'd like to use, you can replace `latest` with that version.
+
+You can also see all available parameters
+[through the web console](https://us-west-2.console.aws.amazon.com/systems-manager/parameters/#list_parameter_filters=Path:Recursive:%2Faws%2Fservice%2Fbottlerocket).
+
+## Launching your first instance
+
+In order to launch a Bottlerocket instance into your ECS cluster, you'll first need some information about the resources in your AWS account.
+
+### Subnet info
+
+You should either have a default virtual private cloud (VPC) or have already
+[created a VPC](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/get-set-up-for-amazon-ecs.html#create-a-vpc)
+in your account.
+
+To find your default VPC, run this command.
+(If you use an AWS region other than "us-west-2", make sure to change that.)
```

Comment:
Instead of giving this warning throughout the doc would it be better to remove `--region` from the example commands and at the top of the doc say they should export `AWS_REGION`? It makes the experience better from a copy/paste workflow and if someone has a default region they don't need to worry about it.

Reply:
I'm planning to leave this as-is for now to keep it consistent with the EKS-focused guide.  We can make larger changes to both guides separately.

(Exported environment variables can also be more confusing to reproduce later; having an explicit command-line option helps for repeatability.)